jax_distribution/wrappers/jumanji.py

```diff
@@ -112,3 +130,72 @@ def observation_spec(self) -> specs.Spec[Observation]:
             "step_count",
         )
         return self._env.observation_spec().replace(step_count=step_count)
+
+
+class RwareMultiAgentWithGlobalStateWrapper(Wrapper):
+    """Multi-agent wrapper for the Robotic Warehouse environment."""
```

Comment:
Just a slightly more descriptive docstring. :) 
```suggestion
    """Multi-agent wrapper for the Robotic Warehouse environment that includes a global environment state
    to be used by the centralised critic. Note here that since robotic warehouse does not have a global state, 
    we create one by concatenating the observations of all agents. 
    """
```